NuGet.Services.BasicSearch.CloudService/ServiceDefinition.csdef

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ServiceDefinition name="NuGet.Services.BasicSearch.CloudService" xmlns="http://schemas.microsoft.com/ServiceHosting/2008/10/ServiceDefinition" schemaVersion="2015-04.2.6">
+  <WebRole name="NuGet.Services.BasicSearch" vmsize="ExtraLarge">
+    <Sites>
+      <Site name="Web">
+        <Bindings>
+          <Binding name="Endpoint1" endpointName="Endpoint1" />
+        </Bindings>
+      </Site>
+    </Sites>
+    <ConfigurationSettings>
+      <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" />
+      <Setting name="Storage.Primary" />
+      <Setting name="Search.IndexContainer" />
+      <Setting name="Search.DataContainer" />
+      <Setting name="Search.IndexRefresh" />
+      <Setting name="Search.RegistrationBaseAddress" />
+    </ConfigurationSettings>
+    <Endpoints>
+      <InputEndpoint name="Endpoint1" protocol="http" port="80" />
```

Comment:
We should always use https. Let's fix this in another merge.

Hui

On Oct 15, 2015, at 11:48 PM, Maarten Balliauw <notifications@github.com<mailto:notifications@github.com>> wrote:


In NuGet.Services.BasicSearch.CloudService/ServiceDefinition.csdef<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fNuGet%2fNuGet.Services.Metadata%2fpull%2f29%23discussion_r42213230&data=01%7c01%7chui.jiang%40microsoft.com%7c25cd2fd0eb4d4de34efc08d2d5f5c7f6%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=Q2SJJmXv%2bydJd3dDHG%2bs22uRyn%2fCnJxsh5ifod9XIME%3d>:

> +      <Site name="Web">
> +        <Bindings>
> +          <Binding name="Endpoint1" endpointName="Endpoint1" />
> +        </Bindings>
> +      </Site>
> +    </Sites>
> +    <ConfigurationSettings>
> +      <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" />
> +      <Setting name="Storage.Primary" />
> +      <Setting name="Search.IndexContainer" />
> +      <Setting name="Search.DataContainer" />
> +      <Setting name="Search.IndexRefresh" />
> +      <Setting name="Search.RegistrationBaseAddress" />
> +    </ConfigurationSettings>
> +    <Endpoints>
> +      <InputEndpoint name="Endpoint1" protocol="http" port="80" />


Should we add an HTTPS binding as well? Would like to see search run over HTTPS where possible.

-
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fNuGet%2fNuGet.Services.Metadata%2fpull%2f29%2ffiles%23r42213230&data=01%7c01%7chui.jiang%40microsoft.com%7c25cd2fd0eb4d4de34efc08d2d5f5c7f6%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=v0Ob8tguVHhnPvxmlMYPSw%2bXVVvogYamKVzWNftJO%2bQ%3d>.